.gitignore

```diff
@@ -0,0 +1,71 @@
+evaluation_experiments/EXP4_cibersort_eval.ipynb
```

Comment:
you might also want to include additional  files that are automatically generated by vscode/python/Mac

link to common files to exclude: https://github.com/microsoft/vscode-python/blob/main/.gitignore

Reply:
nice, added those